Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/Validation/EfficiencyCondition.mo

```diff
@@ -0,0 +1,266 @@
+      color={0,0,127}));
+  connect(con.y, effCon.uCapDes)
+    annotation (Line(points={{-108,80},{-50,80},{-50,6},{-32,6}},
+      color={0,0,127}));
+  connect(con1.y, effCon.uCapUpMin)
+    annotation (Line(points={{-108,40},{-60,40},{-60,3},{-32,3}},
+      color={0,0,127}));
+  connect(conIntp1.y, effCon1.uTyp)
+    annotation (Line(points={{32,-80},{90,-80},{90,-6},{108,-6}},
+      color={255,127,0}));
+  connect(pul2.y, effCon1.VHotWat_flow)
+    annotation (Line(points={{32,0},{72,0},{72,0},{108,0}},
+      color={0,0,127}));
+  connect(pul1.y, effCon1.uCapReq)
+    annotation (Line(points={{32,120},{100,120},{100,9},{108,9}},
+      color={0,0,127}));
+  connect(con2.y, effCon1.uCapDes)
+    annotation (Line(points={{32,80},{90,80},{90,6},{108,6}},
+      color={0,0,127}));
+  connect(con3.y, effCon1.uCapUpMin)
+    annotation (Line(points={{32,40},{80,40},{80,3},{108,3}},
+      color={0,0,127}));
+  connect(conInt.y, effCon.uAvaUp)
+    annotation (Line(points={{-108,-120},{-40,-120},{-40,-9},{-32,-9}},
+      color={255,127,0}));
+  connect(conInt1.y, effCon1.uAvaUp)
+    annotation (Line(points={{32,-120},{100,-120},{100,-9},{108,-9}},
+      color={255,127,0}));
+  connect(con4.y, effCon.VUpMinSet_flow)
+    annotation (Line(points={{-108,-40},{-60,-40},{-60,-3},{-32,-3}},
+      color={0,0,127}));
+  connect(con5.y, effCon1.VUpMinSet_flow)
+    annotation (Line(points={{32,-40},{80,-40},{80,-3},{108,-3}},
+      color={0,0,127}));
+  connect(conIntp2.y,effCon2. uTyp)
+    annotation (Line(points={{172,-80},{230,-80},{230,-6},{248,-6}},
+      color={255,127,0}));
+  connect(con6.y,effCon2. uCapDes)
+    annotation (Line(points={{172,80},{230,80},{230,6},{248,6}},
+      color={0,0,127}));
+  connect(con7.y,effCon2. uCapUpMin)
+    annotation (Line(points={{172,40},{220,40},{220,3},{248,3}},
+      color={0,0,127}));
+  connect(conInt2.y,effCon2. uAvaUp)
+    annotation (Line(points={{172,-120},{240,-120},{240,-9},{248,-9}},
+      color={255,127,0}));
+  connect(con8.y,effCon2. VUpMinSet_flow)
+    annotation (Line(points={{172,-40},{220,-40},{220,-3},{248,-3}},
+      color={0,0,127}));
+  connect(con9.y, effCon.uStaChaProEnd) annotation (Line(points={{-108,-160},{-29,
+          -160},{-29,-12}}, color={255,0,255}));
+  connect(con10.y, effCon1.uStaChaProEnd) annotation (Line(points={{32,-160},{111,
+          -160},{111,-12}}, color={255,0,255}));
+  connect(booPul.y, edg.u)
+    annotation (Line(points={{172,-160},{178,-160}}, color={255,0,255}));
+  connect(edg.y, effCon2.uStaChaProEnd) annotation (Line(points={{202,-160},{251,
+          -160},{251,-12}}, color={255,0,255}));
+  connect(con11.y, effCon2.uCapReq) annotation (Line(points={{172,120},{240,120},
+          {240,9},{248,9}}, color={0,0,127}));
+  connect(con12.y, effCon2.VHotWat_flow)
+    annotation (Line(points={{172,0},{248,0}}, color={0,0,127}));
+  annotation(Icon(coordinateSystem(preserveAspectRatio=false,
```

Comment:
Please don't forget to add revisions section.

Reply:
This applies to all blocks and validation models, please make sure they all have it.